src/routes/metadata.ts

```diff
@@ -7,10 +7,16 @@ import { getMetadataFiles, getNumFramesFromChunkName } from 'util/framekm';
 import { join } from 'path';
 import { promisify } from 'util';
 import { ensureAclPassed } from './middleware/acl';
+import { getConfig } from 'sqlite/config';
 
 const router = Router();
 
 router.get('/', ensureAclPassed, async (req: Request, res: Response) => {
+  if (await getConfig('isLTEUploadEnabled')) {
```

Comment:
I think this validation should be made at getMetadataFiles level, as Im calling the same when Im sending the metadata through the socket (search who is using calling `getMetadataFiles()`)

Reply:
Addressed in https://github.com/Hivemapper/odc-api/pull/245